gear-programs/erc20-relay/app/tests/gclient.rs

```diff
@@ -162,3 +166,105 @@ async fn set_vft_gateway() {
     let vft_gateway = client.vft_gateway().recv(program_id).await.unwrap();
     assert_eq!(vft_gateway, Default::default());
 }
+
+#[tokio::test]
+#[ignore = "Requires running node"]
+async fn test_getters() {
```

Comment:
I think we don't need tests for getters as they're really straightforward

Reply:
The more tests - the better =). Anyway it contains test for `update_config` setter